### PR TITLE
Fix garage deletion

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Fixes the (non) deletion of the garage in local storage. This was caused because of the `robotUpsert` action and the fact that some coordinators took longer to respond, generating the re-creation of the robot.

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: frontend/src/components/BookTable/index.tsx [hunk 1]
```diff
@@ -369,30 +375,33 @@ const BookTable = ({
...
+                size={1.7 * fontSize}
+                text={params.row.payment_method}
+              />
+            </div>
+          );
+        },
+      };
+    },
+    [fav.mode],
+  );
```
Comment (KoalaSat) [inline frontend/src/components/BookTable/index.tsx:404]: I think all modifications on this file was caused by the lintern, I'm pretty sure I didn't change anything here

**Comment by Reckless-Satoshi** [comment]
Thank you for this fix @KoalaSat ! Please submit an invoice for 100K Sats.

**Comment by KoalaSat** [comment]
🎉 
```
lnbc1m1pj6ltkqpp5ljk4rf2g4r0t4qx40afcxh3mfcfw8ahkwy5rt2jvgxn4qmcl6gksdqqcqzzsxqyz5vqsp5epv8slxguugu3lde7l24sx3q5znf55vxeul7mrjd5pk8ua7pnems9qyyssqrf6xs0y0tumferdkxffefzgf6y0cv472eqfee5uhfe6ax3uz2w6ngpmf3n8z70yxdu8p2rs9lc230sp48s2k3vh49r9a07nzw56qyqgpzlqdah
```

**Comment by Reckless-Satoshi** [comment]
> lnbc1m1pj6ltkqpp5ljk4rf2g4r0t4qx40afcxh3mfcfw8ahkwy5rt2jvgxn4qmcl6gksdqqcqzzsxqyz5vqsp5epv8slxguugu3lde7l24sx3q5znf55vxeul7mrjd5pk8ua7pnems9qyyssqrf6xs0y0tumferdkxffefzgf6y0cv472eqfee5uhfe6ax3uz2w6ngpmf3n8z70yxdu8p2rs9lc230sp48s2k3vh49r9a07nzw56qyqgpzlqdah

e9df8e357f05fd43d47cb71b104859ca0d470cb745d8e446ad0540c0c061ca39

**Pull request merged**